verilog/tools/ls/symbol-table-handler.h

```diff
@@ -48,6 +48,20 @@ class SymbolTableHandler {
   // Once the project's root is set, a new SymbolTable is created.
   void SetProject(const std::shared_ptr<VerilogProject> &project);
 
+  // prepares structures for symbol-based requests
+  void Prepare();
```

Comment:
Does this need to be public ? We should be very specific in what should be visible in the API.
If there are functions needed in tests, it is ok to make public, but add a comment.

Reply:
I made all internally used functions `private` for now (except for methods used in AUTO expansions).